apps/condo/domains/meter/schema/MeterReading.js

```diff
@@ -123,6 +130,24 @@ const MeterReading = new GQLListSchema('MeterReading', {
 
             return resolvedData
         },
+        validateInput: async ({ context, resolvedData, existingItem }) => {
+            const newItem = { ...existingItem, ...resolvedData }
+            const meterId = get(newItem, 'meter')
+
+            const meter = await Meter.getOne(context, {
```

Comment:
Use `getById` or `getByCondition`

Reply:
You don't need to fetch  all relations just to extract number of tariffs

Reply:
Okay, fixed